### PR TITLE
Completed unit-tests-jest exercise

### PR DESCRIPTION
**Pull request by tam-n**
image

### CODE REVIEW

File: unit-tests-jest/numbers.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ * @param {Array} numbers The array of numbers to be divided by the divisor.
+ * @param {number} divisor The number to divide into each element of `numbers`.
+ * @returns a new array.
+ */
+export function divideBy(numbers, divisor) {
+  const newArray = numbers.map((number) => {
+    if (number === 0 && Math.sign(divisor) === -1) {
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.js:28]: I don't understand why you are doing this check.

File: unit-tests-jest/numbers.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:2]: These are good happy path tests. You should also test the edge cases.

**Review by RobertGardner — APPROVED** [review]
